TLM/TLM/Lifecycle/TMPELifecycle.cs

```diff
@@ -327,13 +245,12 @@ public class TMPELifecycle : MonoBehaviour {
                     ModUI.Instance.Destroy();
                     Log._Debug("removed UIBase instance.");
                 }
-            } catch (Exception e) {
-                Log.Error("Exception unloading mod. " + e.Message);
 
-                // ignored - prevents collision with other mods
+                Patcher.Uninstall();
```

Comment:
Exception in any code above will prevent uninstalling patches -> game will be broken